components/RemedyForm.js

```diff
@@ -0,0 +1,169 @@
+import { useState } from "react";
+import symptoms from "../assets/symptoms.json";
+import styled from "styled-components";
+
+export default function RemedyForm({ onAddRemedy }) {
+  const [ingredients, setIngredients] = useState([""]);
+
+  const [selectedSymptoms, setSelectedSymptoms] = useState([]);
+
+  function handleIngredientChange(index, value) {
+    const newIngredients = [...ingredients];
+    newIngredients[index] = value;
+    setIngredients(newIngredients);
+  }
+
+  function handleAddIngredients() {
+    setIngredients([...ingredients, ""]);
+  }
+
+  function handleRemoveIngredients(index) {
+    const newIngredients = ingredients.filter(
+      (_, currentIndex) => currentIndex !== index
+    );
+    setIngredients(newIngredients);
+  }
+
+  function handleSelectSymptom(event) {
+    const { value } = event.target;
+    if (value && !selectedSymptoms.includes(value)) {
+      setSelectedSymptoms([...selectedSymptoms, value]);
+    }
+  }
+
+  function handleRemoveSymptom(index) {
+    const newSymptoms = selectedSymptoms.filter(
+      (_, currentIndex) => currentIndex !== index
+    );
+    setSelectedSymptoms(newSymptoms);
+  }
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+    const newRemedy = {
+      title: formData.get("title"),
+      ingredients: ingredients,
+      preparation: formData.get("preparation"),
+      usage: formData.get("usage"),
+      symptoms: selectedSymptoms,
+      imageUrl: "/placeholder.jpg",
+    };
+    onAddRemedy(newRemedy);
+    event.target.reset();
+    event.target.title.focus();
+    setIngredients([""]);
+    setSelectedSymptoms([]);
+  }
+
+  return (
+    <Form onSubmit={handleSubmit}>
+      <h2>Add Remedy</h2>
+
+      <label htmlFor="title">
+        Title:<span aria-label="required">*</span>
+      </label>
+      <input
+        id="title"
+        name="title"
+        type="text"
+        placeholder="Enter remedy title"
+        required
+      />
+
+      <section>
+        <label htmlFor="ingredients-group">
+          Ingredients:<span aria-label="required">*</span>
+        </label>
+        {ingredients.map((ingredient, index) => (
+          <div key={index}>
+            <input
+              id={`ingredient-${index}`}
+              type="text"
+              name="ingredients"
+              value={ingredient}
+              placeholder="Enter remedy ingredient"
+              onChange={(event) =>
+                handleIngredientChange(index, event.target.value)
+              }
+              required
+            />
+
+            {ingredients.length > 1 && (
+              <button
+                type="button"
+                onClick={() => handleRemoveIngredients(index)}
+              >
+                🗑️
+              </button>
+            )}
```

Comment:
It is advisable to wrap icons in a span element and use an aria-label to make the button's purpose clear to screen reader users.

```suggestion
             <button type="button" onClick={() => handleRemoveIngredients(index)}>
           <span aria-label="Remove ingredient">🗑️</span>
           </button>
            )}
```